docs/index.html

```diff
@@ -181,14 +183,10 @@
 evaluates to the `integer` `3`.  The `{}` block ensures that `one` and `two`
 are not accessible after `three` is computed.
 
-```reason
-let three = {
-  let one = 1;
-  let two = 2;
-  one + two
-};
-/* one and two not accessible here! */
-```
+<pre>
+<p data-id="-P8x86Mg6w" data-host="replay-154206.appspot.com" data-height="256" data-font-size="12px" class="replay" ></p>
```

Comment:
@yutongp Thanks, this makes sense. Adding original code snippet in the comment looks good. It's a bit more effort to maintain all the "dataid" here, but I think in the short term, we can always require contributors who changes the comment here to change the "dataid" as well. 

In the long term, we should put the comments into separate files, and have a script to automatically generate the code snippet id and master html file based on those snippet files.